src/test/java/agh/ics/oop/IntegrationTests.java

```diff
@@ -0,0 +1,56 @@
+package agh.ics.oop;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class IntegrationTests {
+    @Test
+    void movingTest1() {
```

Comment:
ok, przy czym warto byłoby ubrać to w jakieś konkretne przypadki testowe, test 1 i test 2 średnio mówią co tu jest robione (i czemu są akurat 2 a nie np. 3 albo 4 :P)